arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -2026,7 +2033,8 @@ Result RocksDBCollection::lookupDocumentVPack(
 }
 
 void RocksDBCollection::setupCache() const {
-  if (_cacheManager == nullptr || !_cacheEnabled) {
+  if (_cacheManager == nullptr ||
+      !_cacheEnabled.load(std::memory_order_relaxed)) {
```

Comment:
```suggestion
      !_cacheEnabled.load(std::memory_order_relaxed)) {
```
Is this condition assert? I don't see the place that doesn't look like
if (cache enabled) setupCache